examples/explainable_ai/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+
+pre_run () {
+  pip install torchvision
+  zenml integration install pytorch foxai
```

Comment:
Maybe I'm missing something, but there currently is no `foxai` integration in ZenML right? Are you planning to add this as part of this PR?

Reply:
Or should this be a `pip install foxai` instead?

Reply:
For now, I can remove this.